roaster-tm4j/roaster-tm4j-common/src/main/java/hu/icellmobilsoft/roaster/tm4j/common/Tm4jReporterProducer.java

```diff
@@ -0,0 +1,56 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.tm4j.common;
+
+import hu.icellmobilsoft.roaster.tm4j.common.api.reporter.TestResultReporter;
+import hu.icellmobilsoft.roaster.tm4j.common.config.Tm4jReporterConfig;
+
+import javax.enterprise.context.Dependent;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+/**
+ * Creates a {@code TestResultReporter} based on the Roaster configuration.
+ *
+ * @author martin.nagy
+ * @since 0.2.0
+ */
+@Dependent
```

Comment:
Producer osztalyok altalaban ApplicationScoped, de igy is jo. Valoszinu a 39-41 sor miatt raktad dependent-re, de mukodik...